src/js/components/MarathonTaskDetailsList.js

```diff
@@ -58,6 +58,14 @@ class MarathonTaskDetailsList extends React.Component {
     return task.status;
   }
 
+  getTimeField(time) {
+    return (
+      <time dateTime={time} title={time}>
+        {new Date(time).toLocaleString()}
```

Comment:
Should we be using locale strings here, or does it make more sense to use an ISO string? Or do we want something from `moment`?